test/timeout_test.go

```diff
@@ -84,20 +84,20 @@ func TestPipelineRunTimeout(t *testing.T) {
 	defer close(errChan)
 
 	for _, taskrunItem := range taskrunList.Items {
-		go func() {
-			err := WaitForTaskRunState(c, taskrunItem.Name, func(tr *v1alpha1.TaskRun) (bool, error) {
+		go func(name string) {
```

Comment:
and here 
```suggestion
		wg.Add()
		go func(name string) {
			defer wg.Done()
```

with a `wg.Wait()` outside the loop

Reply:
oh wait we don't need for those ones I think